tools/benchcomp/configs/perf-regression.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# Run the Kani perf suite twice, erroring out on regression. This config
+# file is primarily intended to be used in CI, because it assumes that
+# there are two Kani checkouts in the 'base' and 'pr' directories;
+# benchcomp compares the performance of these two checkouts.
+
+variants:
+  kani_pr:
+    config:
+      directory: pr
+      command_line: PATH=$(realpath pr/scripts):$PATH scripts/kani-perf.sh
+      env:
+        RUST_TEST_THREADS: "1"
+  kani_base:
+    config:
+      directory: base
+      command_line: PATH=$(realpath base/scripts):$PATH scripts/kani-perf.sh
+      env:
+        RUST_TEST_THREADS: "1"
+
+run:
+  suites:
+    kani_perf:
+      parser:
+        module: kani_perf
+      variants: [kani_base, kani_pr]
+
+visualize:
```

Comment:
The `error_on_regression` visualization does print out any benchmark that regressed as a warning to stderr.

It doesn't attempt to print anything out in a pretty/parseable way. The idea would be that we'd write a different visualization to do that. This is because you might want it to print something else to stdout, e.g. the `dump_yaml` visualization or a different kind of visualization.